docs/installation/gcp.md

```diff
@@ -0,0 +1,227 @@
+---
+sidebar_position: 4
+---
+
+# GCP Install Guide
+
+## Subsratus required resources
+* GKE Cluster with GCSfuse and Workload Identity
+* GCS Bucket to store Model and Dataset data
+* Google Artifact Registry Container Repository to store newly built container images
+* Google Service Account for accessin GCS Bucket, Artifact Registry and managing cross
```

Comment:
`accessin` --> `read/write access to`

Reply:
I would use sub-bullets for the access that this SA will need:

* Read/Write: Objects within Google Cloud Storage Bucket
* Read/Write: Artifacts within Google Artifact Registry
* Read/Write: Workload Identity bindings

Reply:
done